apps/forms/base.py

```diff
@@ -124,27 +124,22 @@ def get_common_field(self, field_name: str, spinner: bool = False, **kwargs):
         help_message attribute to it. The template then uses it to render the tooltip for all fields
         using this class.
         """
-        if spinner is True:
-            template = "apps/custom_field_with_spinner.html"
-            base_args = dict(
-                css_class="form-control form-control-with-spinner",
-                wrapper_class="mb-3",
-                rows=3,
-                help_message=HELP_MESSAGE_MAP.get(field_name, ""),
-            )
-        else:
-            template = "apps/custom_field.html"
-            base_args = dict(
-                css_class="form-control",
-                wrapper_class="mb-3",
-                rows=3,
-                help_message=HELP_MESSAGE_MAP.get(field_name, ""),
-            )
+
+        spinner = kwargs.pop("spinner", False)
+
+        template = "apps/custom_field.html"
+        base_args = dict(
+            css_class="form-control form-control-with-spinner" if spinner else "form-control",
+            wrapper_class="mb-3",
+            rows=3,
+            help_message=HELP_MESSAGE_MAP.get(field_name, ""),
+            spinner=spinner,
+        )
 
         base_args.update(kwargs)
         field = CustomField(field_name, **base_args)
         field.set_template(template)
-        return Div(field)
+        return Div(field, css_class="form-input-with-spinner" if spinner else None)
```

Comment:
No need to double wrap the field in a Div, so just add the `form-input-with-spinner` css class here if `spinner` is `True`